snap/snapcraft.yaml

```diff
@@ -1,4 +1,4 @@
-name: spark-k8s # you probably want to 'snapcraft register <name>'
+name: spark-k8s
 base: core20
 version: '0.1'
 summary: Client side snap to submit spark jobs to kubernetes.
```

Comment:
Please rephrase this as "Client side scripts to submit Spark jobs to a cluster"

Reply:
at line 30 (reviewing tool is broken fro some reason), please use a dedicated mirror of spark with the `pom.xml` adapted to integrate the dependencies, rather than using wget

Reply:
Summary rephrased. Regarding building from pom.xml, after discussion working with binaries for now.